packages/react-ui-components/src/MultiSelectBox_ListPreviewSortable_DraggableListPreviewElement/multiSelectBox_ListPreviewSortable_DraggableListPreviewElement.js

```diff
@@ -129,7 +137,11 @@ export default class MultiSelectBox_ListPreviewSortable_DraggableListPreviewElem
                             hoverStyle={'clean'}
                             />
                     )}
-                    <div className={theme.selectedOptions__innerPreview}>
+                    <div
+                        className={theme.selectedOptions__innerPreview}
+                        onClick={this.handleClick}
```

Comment:
```suggestion
                        onClick={this.props.onItemClick ? this.handleClick : null}
```

Reply:
Outside the `ReferencesEditor` use case, most instances of the `<MultiSelectBox/>` have no way of informing this click handler. It should therefore only be set, if it can actually perform an action.